orchestrator-salt/src/main/resources/salt/salt/cloudera/manager/init.sls

```diff
@@ -44,7 +44,10 @@ add_settings_file_to_cfm_server_args:
   file.replace:
     - name: /etc/default/cloudera-scm-server
     - pattern: "CMF_SERVER_ARGS=.*"
-{% if salt['pillar.get']('cloudera-manager:settings:set_cdp_env') == True %}
+{% if salt['pillar.get']('cloudera-manager:settings:cloud_provider_setup_supported') == True %}
+    - repl: CMF_SERVER_ARGS="-i /etc/cloudera-scm-server/cm.settings -cp {{ salt['pillar.get']('platform') }} -env PUBLIC_CLOUD"
+    - unless: grep "CMF_SERVER_ARGS=\"-i /etc/cloudera-scm-server/cm.settings -cp {{ salt['pillar.get']('platform') }}\"" /etc/default/cloudera-scm-server
+{% elif salt['pillar.get']('cloudera-manager:settings:set_cdp_env') == True and salt['pillar.get']('cloudera-manager:settings:cloud_provider_setup_supported') == False %}
```

Comment:
`and salt['pillar.get']('cloudera-manager:settings:cloud_provider_setup_supported') == False` this part is not required

Reply:
Addressed, plz review